app/helpers/feedback_helper.rb

```diff
@@ -1,6 +1,6 @@
 module FeedbackHelper
   def referrer_is_claim?(referrer)
-    referrer =~ /claims/
+    referrer.include? 'claims'
```

Comment:
I'd put a safe navigation operator on here to emulate current potential behaviour
```suggestion
    referrer&.include? 'claims'
```

```
referrer = nil
referrer =~ /claims/
=> nil
referrer.include?('claims')
=> NoMethodError: undefined method `include?' for nil:NilClass
referrer&.include?('claims')
=> nil
```

Reply:
Seems sensible. Probably good for the other case of this too, in `lib/rack/injectable.rb`.